### PR TITLE
Proper comment detection

### PR DESCRIPTION
**Pull request by paulpflug**
I'm not sure what `getSerializedParams` should do, but it fails quite often for me...

### CODE REVIEW

File: lib/coffee-autocompile.coffee [hunk 1]
```diff
@@ -19,9 +19,18 @@ class StylusAutocompile
     @compile()
 
   compile: ->
-    params = @getParams @activeEditor.getURI(), @activeEditor.getText()
-    return unless params?
-
+    text = @activeEditor.getText()
+    firstComment = text.match /^\s*#\s*(.*)\n*/
```
Comment (olmokramer) [inline lib/coffee-autocompile.coffee]: This doesn't seem to check for a hashbang (`#!...`) that *must* be on the first line anymore

**Comment by olmokramer** [comment]
Thanks for the PR :) one slight issue, though. Check the comment on line 23

File: lib/coffee-autocompile.coffee (hunk 1 above)
Comment (olmokramer) [inline lib/coffee-autocompile.coffee]: This one should do: `/^(?:#!.*\n)?\s*#\s*(.*)\n*/`. I'll merge this and fix the regex.

**Comment by olmokramer** [comment]
Ok I published a new version that includes this patch and a few other fixes. Cheers

**Comment by paulpflug** [comment]
cool :+1:

**Pull request merged**